tests/autoyast/installation.pm

```diff
@@ -205,6 +215,12 @@ sub run {
                 send_key_until_needlematch 'create-partition-plans-finished', $cmd{ok};
                 next;
             }
+            for (@expected_warnings) {
+                if (match_has_tag $_) {
+                    push(@actual_warnings, $_);
```

Comment:
if more than one warning I guess you would like to remove `$_` from `@expected_warnings` to not compare twice and at the end instead of comparing both arrays, just check for size 0.